dev/lnd/lnd.conf

```diff
@@ -25,6 +25,8 @@ bitcoin.active=true
 bitcoin.defaultchanconfs=0
 bitcoin.node=bitcoind
 bitcoin.regtest=true
+; bitcoin.minhtlc=1
+bitcoin.minhtlcout=1
```

Comment:
Got help from Alex to eventually figure this setting out to be able to do the `995 msats` integration test

Reply:
we could also set the minhtlc to 1000 msats?

I'm not sure whether we need to support sub sats payment use cases.